### ISSUE TITLE
LM40SAMFHD700AG25WV panel. so confused. please help.

### ISSUE THREAD

**Issue by tung256** [olm3ca/mirror#15]
the back panel has 2 cables going in:
![image](https://github.com/user-attachments/assets/a9b8dbf4-9361-4ee6-9921-3eac3d8c19d5)

in one corner, i see this sticker
![image](https://github.com/user-attachments/assets/eb041299-257f-473f-8e2e-ac9c67d413cb)

i believe this is the new controller board that i need: https://www.ebay.com/itm/235438526466?var=535565344805

i see the OLD and NEW dont match, not even same colors! can i just plug the new cable in?
![image](https://github.com/user-attachments/assets/ebb1716d-2c6a-4765-8f52-ef788d2415c7)

here is the NEW controller with all the cables 
![image](https://github.com/user-attachments/assets/dfc571c4-da7b-4bf9-ae3e-8a04f8bf3a97)

here's a close up of the NEW:
![image](https://github.com/user-attachments/assets/8b7b6716-4e1c-4230-a128-c327653e702e)

and the back of the NEW
![image](https://github.com/user-attachments/assets/ea675b7c-ccf0-4bb6-90a8-5535b2032222)

here is the OLD board along with some of the old cables
![image](https://github.com/user-attachments/assets/1c8e77bd-0fb3-4728-8447-1f9686adc2fd)

what is the A cable for? B is labeled as AUDIO HDR, how can i connect to the NEW board?
C is power. how to use this C  to power up the NEW board?

do I ignore D cable?  it is the other cable that goes to the panel which i have no idea what it's for....

please help. and if you have a video of it explaining all the connections, kindly share as well. that would help greatly!

**Comment by jeremy-fields** [comment]
I think [this](https://github.com/olm3ca/mirror/issues/10#issuecomment-2076426623) has a lot of your answers in it. I pulled a lot of cables out when I removed the old board. That board was connected to a lot including a camera, microphone, and speakers. The new board can power speakers, but the connectors are different. I did not connect audio in mine, but someone else did. They talk about it [here](https://github.com/olm3ca/mirror/issues/8#issuecomment-2094972987).
